examples/pitch-field-demo/field_pitch.js

```diff
@@ -91,10 +91,10 @@ CustomFields.FieldPitch.prototype.showEditor_ = function() {
   // change this behaviour.  For now, using bindEvent_ instead of
   // bindEventWithChecks_ allows it to work without a mousedown/touchstart.
   this.clickWrapper_ =
-      Blockly.bindEvent_(this.imageElement_, 'click', this,
```

Comment:
We actually left aliases for these, so the renamings database needs to be modified.